tunnel/intercept/tproxy/tproxy_linux.go

```diff
@@ -613,6 +613,19 @@ func (self *tProxy) StopIntercepting(tracker intercept.AddressTracker) error {
 			if err != nil {
 				errorList = append(errorList, err)
 				log.WithError(err).Errorf("failed to remove route %v for service %s", ipNet, *self.service.Name)
+			} else {
+				host, hostErr := self.resolver.Lookup(ipNet.IP)
+				if hostErr == nil {
+					hostErr = self.resolver.RemoveHostname(host)
+					if hostErr == nil {
+						pfxlog.Logger().Debug("Removed Hostname from Resolver%v", host)
```

Comment:
we should use the same log technique as other messages in this function. e.g.:
```
log.Debugf("Removed Hostname from Resolver %v", host)
```
same thing on line 623 below.

Reply:
ok thanks.  changes pushed.